### PR TITLE
CODESEE-2183 Allow multiple maps on OSS page

### PR DESCRIPTION
**Pull request by joshuadoan**
Updated `readme` 

If you would like to display multiple maps you can add an entry called "maps"

```
maps:
  - url: https://app.codesee.io/maps/public/848e3630-1650-11ec-8bc1-7d4a4822cc27
    description: Get a quick visual overview of the major areas of our repo!
    subTitle: devdocs
  - url: https://app.codesee.io/maps/public/848e3630-1650-11ec-8bc1-7d4a4822cc27
    description: Another map!
    subTitle: testMap
```
OSS Port OSS Port 2021-11-03 at 2 41 26 PM

### CODE REVIEW

File: src/components/MapsSection.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        <ul className="md:flex gap-2 m-0 p-0">
+          {
+            maps.map(({ url, description, subTitle }, i) => (
+              <li key={`map-${i}`} style={{
+                width: 300
+              }}
+                className="p-4 bg-white mb-2">
+                <div className="flex justify-center">
+                  <FormattedLink href={url} >
+                    <Logo />
```
Comment (deammer) [inline src/components/MapsSection.tsx:25]: **suggestion (non-blocking):** the metadata endpoint returns a thumbnail, so I think you could display it here instead?

**Review by deammer — APPROVED** [review]
Perfect! Just one non-blocking suggestion.

File: gatsby-node.js [hunk 2]
```diff
@@ -107,6 +107,11 @@ exports.createPages = async ({ actions, graphql, reporter, cache }) => {
               title
               url
             }
+            maps {
```
Comment (LizardEatsFlies) [inline gatsby-node.js:110]: adding a comment

File: src/types.ts [hunk 3]
```diff
@@ -22,6 +22,11 @@ export type ProjectFrontmatter = {
     url: string;
     description: string;
   };
+  maps?: {
+    url: string;
```
Comment (LizardEatsFlies) [inline src/types.ts:26]: yeah this looks great, comment maybe?

**Review by LizardEatsFlies — COMMENTED** [review]
please make changes (this is just a demo)

**Pull request merged**